### PR TITLE
[JN-477] admin help for export

### PR DESCRIPTION
**Pull request by devonbush**
This adds basic help docs (copied over from Pepper) for the download capability.  The existence of help docs within our app is probably temporary.  Long-term, we'll want to use ZenDesk or some other solution that lets non-engineers write/edit the docs.   I'll put in a request to comms to see about getting that set up.  In the meantime, this will give us a place to park user-visible documentation.  Also, having it be in-app makes the release/review process much simpler -- the docs get auto-released with the features, rather than having to be reviewed and released separately.

image

image

TO TEST:
1. Go to https://localhost:3000/ourhealth/env/live/participants
2. click "download"
3. click the link to the help page
4. confirm the help page shows up in a new tab

### CODE REVIEW

File: ui-admin/src/help/ExportHelp.tsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+      <ul>
+        <li>
+          <b>.xlsx</b> will create an Excel spreadsheet of the participant data. Empty cells will represent
+                    null values.
+        </li>
+        <li>
+          <b>.tsv</b> (tab-delimited values). Will export a tab-delimited file. This may be useful in
+                    environments where Excel is unavailable, or if the
+                    number of columns to be exported exceeds 16K. In order to have data be compliant, double-quotes will
+                    be replaced by single quotes, and any values including
```
Comment (breilly2) [inline ui-admin/src/help/ExportHelp.tsx:22]: It isn't really necessary to replace double quotes with single quotes. It's unlikely that the meaning of any responses will be different, but I have to confess to being a little uneasy about any modification of responses.

File: ui-admin/src/study/participants/export/ExportDataControl.test.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  expect(screen.getByText('Excel (.xlsx)')).toBeInTheDocument()
+})
+
+test('help page loads', async () => {
+  const { RoutedComponent } = setupRouterTest(
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    <ExportDataControl studyEnvContext={mockStudyEnvContext()} show={true} setShow={() => {}}/>)
+  render(RoutedComponent)
+  userEvent.click(screen.getByText('help page'))
+  waitFor(() => expect(screen.getByText('Participant List Export Info')).toBeInTheDocument())
```
Comment (breilly2) [inline ui-admin/src/study/participants/export/ExportDataControl.test.tsx:24]: Interesting. I didn't expect to not need any code to recognize the content opened in a new window.

**Review by breilly2 — APPROVED** [review]

File: ui-admin/src/help/ExportHelp.tsx (hunk 1 above)
Comment (devonbush) [reply ui-admin/src/help/ExportHelp.tsx:22]: you're right that they can be escaped, but other applications handling of escaped quotes in csv files varies enough that it was decided for Pepper that this strategy would result in less user issues (and they can always grab the .xlsx format if quote fidelity is important)

File: ui-admin/src/study/participants/export/ExportDataControl.test.tsx (hunk 2 above)
Comment (devonbush) [reply ui-admin/src/study/participants/export/ExportDataControl.test.tsx:24]: I'm guessing that the headless browser just ignores the target="_blank" and opens it in the same window

**Review by MatthewBemis — APPROVED** [review]

**Pull request merged**